src/org/usfirst/frc/team2706/robot/subsystems/DriveTrain.java

```diff
@@ -26,6 +29,11 @@
 	private Encoder left_encoder, right_encoder;
 	private AnalogInput rangefinder;
 	private AnalogGyro gyro;
```

Comment:
Yes. Unfortunately, it sounds like we will also have to worry about supporting another gyro type as well.